tools/star.wdl

```diff
@@ -160,6 +180,6 @@ task alignment {
         read_one_fastqs: "An array of FastQ files containing read one information"
         read_two_fastqs: "An array of FastQ files containing read two information in the same order as the read one FastQ"
         stardb_tar_gz: "A gzipped TAR file containing the STAR reference files"
-        read_groups: "A string containing the read group information to output in the BAM file"
+        read_groups: "A string containing the read group information to output in the BAM file. Read group fields (Required fields: ID, LB, PL, PU, & SM) should be space delimited. Read groups should be comma separated, with a space on each side (e.g. ' , '). The ID for each read group must match the basename of a fastq file (up to the first period). Expected form: `ID:rg1 PU:flowcell1.lane1 SM:sample1 PL:illumina LB:sample1_lib1 , ID:rg2 PU:flowcell1.lane2 SM:sample1 PL:illumina LB:sample1_lib1`"
```

Comment:
These are only required fields when part of RNA-Seq V2. At the task level, the only requirement is `ID:<fastq-basename matching string>[ , ID:<other fastq basename>...]`. 

I still think having the example in both will be good, but I'd switch it from `Expected form:` to `Example:` here, as the requirements are looser.

Reply:
Picard ValidateSam requires all of those fields, so it will pass the sort step, but will error out on the Picard validate step.

Reply:
But it's possible (probably only with an external user workflow) to be running the STAR task without the `ValidateSam` step. IMO we should only document what's needed for the specific task, and move those inter-connected requirements to the workflow level. We've already made it optional which would error on `ValidateSam`.

Sidenote: is it a Picard warning or error? Don't we have it strict on warnings by default, but configurable to ignore them? If a user doesn't want to fill in proper dummy values, can they disable an option to get past the validation step?

Reply:
Picard errors on missing read group fields e.g. `MISSING_PLATFORM_VALUE`. 

I'll swap the language to "Example" as you point out, this can be run without the downstream validation.